benchmark/test/test_httpapi.py

```diff
@@ -57,7 +60,8 @@ class BenchmarkAPITests(TestCase):
     # because we test HTTP requests via an actual TCP/IP connection.
     run_tests_with = AsynchronousDeferredRunTest.make_factory(timeout=1)
 
-    RESULT = {'branch': 'branch1', 'run': 1, 'result': 1}
+    RESULT = {u"userdata": {u"branch": "master"}, u"run": 1, u"result": 1,
+              u"timestamp": datetime.now().isoformat(), }
```

Comment:
Is the use of the `datetime.now()` important? If we just need a date for the timestamp field, it should probably be fixed like the later timestamp values.